include/swift/AST/NameLookupRequests.h

```diff
@@ -164,9 +166,10 @@ class SuperclassDeclRequest :
 };
 
 class InheritedProtocolsRequest
-    : public SimpleRequest<InheritedProtocolsRequest,
-                           ArrayRef<ProtocolDecl *>(ProtocolDecl *),
-                           CacheKind::SeparatelyCached> {
+    : public SimpleRequest<
+          InheritedProtocolsRequest, ArrayRef<ProtocolDecl *>(ProtocolDecl *),
+          CacheKind::SeparatelyCached | CacheKind::DependencySink |
```

Comment:
Sounds like CacheKind is no longer a great name. Perhaps just RequestFlags or something?

Reply:
Done.